fe/fe-core/src/main/java/com/starrocks/common/GenericPool.java

```diff
@@ -65,21 +65,32 @@ public GenericPool(String className, GenericKeyedObjectPoolConfig config, int ti
 
     public boolean reopen(VALUE object, int timeoutMs) {
         boolean ok = true;
-        object.getOutputProtocol().getTransport().close();
+        try {
+            object.getOutputProtocol().getTransport().close();
```

Comment:
Do we need to check the error `java.net.SocketException: Socket closed`?

Reply:
I don’t think it’s necessary, No matter what error is reported by close(), we should go to the open() function.

Reply:
What the errors could return by the close()?